### PR TITLE
Apply fix for deleted item record

### PR DESCRIPTION
**Pull request by rfox2**
**What** Bug fix for DEC-237 - relational integrity issue when item deleted that is attached to a showcase
**Why** It's a bug!
**How** Created a conditional in the relevant view that tests for the existence of the item in question

### CODE REVIEW

File: app/views/sections/_image_form.html.erb [hunk 1]
```diff
@@ -1,11 +1,13 @@
 <div class="row">
   <div class="col-md-9">
     <%= f.input :caption, input_html: { class: 'honeycomb_redactor' }, as: :text %>
-  </div>
+</div>
```
Comment (jaronkk) [inline app/views/sections/_image_form.html.erb]: This closing tag needs to be indented two spaces

File: app/views/sections/_image_form.html.erb [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 
-  <div class="col-md-3">
-    <%= HoneypotThumbnail.display(f.object.item.honeypot_image) %>
-  </div>
+<div class="col-md-3">
+    <% if f.object.item %>
+      <%= HoneypotThumbnail.display(f.object.item.honeypot_image) %>
+    <% end -%>
+</div>
```
Comment (jaronkk) [inline app/views/sections/_image_form.html.erb]: The open and close tag for `<div class="col-md-3">` need to be indented two spaces

File: app/views/sections/_image_form.html.erb (hunk 1 above)
Comment (rfox2) [inline app/views/sections/_image_form.html.erb]: Done

File: app/views/sections/_image_form.html.erb (hunk 2 above)
Comment (rfox2) [inline app/views/sections/_image_form.html.erb]: Done

**Comment by jaronkk** [comment]
:+1:

**Comment by jonhartzler** [comment]
:+1:

**Pull request merged**